### DISCUSSION TITLE
Sounds of game - zooming out the view in fact doesn't reduce the volume of sounds

### DISCUSSION THREAD

**Discussion by LC-Zorg** [OpenTTD/OpenTTD#11729] [Features]
## Version of OpenTTD
1.11.0 and previous
## Expected result
Zooming out the view reduces sounds appropriately, and maximal zooming out turns them off completely or almost completely.
## Actual result
Zooming out reduces the volume of sounds, but not much. Meanwhile, because the distant view covers a larger area, the game generates much more noise. As a result, zooming out of view increases noise instead of reducing it.
## Steps to reproduce
- For example, run this save ([opentitle 2021](https://www.tt-forums.net/download/file.php?id=216853))
- turn on game sounds
- listen to the sounds for a moment, pay attention to the volume
- zoom out the view - the number of sounds will increase, but their volume will not change much and the total volume will increase
## Comments
The solution to the problem may have something to do with #8950

**Comment by PeterN** [comment]
Three things going on here.

1. Zooming out doesn't currently affect already playing sounds, only new sounds.
2. Volume scaling factor for each zoom level is not very strong.
3. As reported, large viewports produce almost constant sounds.

I've already been looking at 1 and 2. Not sure about the best way to tackle 3 though.

**Comment by LC-Zorg** [comment]
What if you link the volume level with the distance level?

Multiplier of current volume setting:
Zoom x4: 100%
Zoom x2: 90%
Zoom x1: 80%
Zoom x1/2: 60%
Zoom x1/4: 40%
Zoom x1/8: 20%
